docs/source/api.rst

```diff
@@ -125,3 +125,67 @@ For example::
 
 
 
+Detailed API reference
+------------------------
+Module: :mod:`traitlets.traitlets`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. automodule:: traitlets.traitlets
+    :members:
+    :private-members:
+
+Module: :mod:`traitlets.config.configurable`
```

Comment:
In that case, we may need to not use automodule, and instead use:

```python
.. module:: traitlets.config

.. autoclass:: Configurable
```

It is important that these be documented under the correct module to import from.